src/tests/functional/plugin/shared/src/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -44,6 +44,22 @@ void OVPropertiesTests::TearDown() {
     }
 }
 
+std::string OVSetPropComplieModleGetPropTests::getTestCaseName(testing::TestParamInfo<CompileModelPropertiesParams> obj) {
+    std::string device_name;
+    AnyMap properties;
+    AnyMap compileModelProperties;
+    std::tie(device_name, properties, compileModelProperties) = obj.param;
+    std::ostringstream result;
+    result << "device_name=" << device_name << "_setProperties_compileModelProperties_getProperties";
```

Comment:
the case name is not unique here, better to combine with input parameters

Reply:
done